js/functions.js

```diff
@@ -630,4 +630,20 @@ function setupMobileTicker() {
 			startEmpty:true
 		});
 		
+}
+
+// Parses passed url and looks for the form 'https://:34343/something'. If after double slash after protocol there is no host present, a local host is assumed and is placed before port part.
+// The url from the example will be transformed into 'https://somehost:34343/something', where somehost is the host at which the muximux was accessed.
+function parseRelativeUrlWithPortIfPresent(url) {
+    var parts = /^(http|https|ftp):\/\/:(\d+)(.*)$/.exec(url),
```

Comment:
Huh. That's a long list. :) I think `^([a-z][a-z0-9+\-.]*):` regexp for [URL Scheme](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax) will work better there. ;)